zml/tokenizer.zig

```diff
@@ -120,16 +122,69 @@ pub const Tokenizer = struct {
         return self.token_lookup.get(str);
     }
 
+    fn toHex(dst: []u8, src: []const u8, comptime case: std.fmt.Case) ![]u8 {
+        return std.fmt.bufPrint(dst, "<0x{s}>", .{switch (case) {
+            .lower => std.fmt.fmtSliceHexLower(src),
+            else => std.fmt.fmtSliceHexUpper(src),
```

Comment:
in general switch/else should be avoided. please explicit all cases as much as possible.

Reply:
Will do